av/sidedata/motionvectors.pyi

```diff
@@ -0,0 +1,25 @@
+from typing import Any, Sequence
+
+import numpy as np
+
+from .sidedata import SideData
+
+class _MotionVector(SideData):
```

Comment:
_MotionVector is not supposed to be part of the Public API, so it's attributes should be merged with MotionVector